### ISSUE TITLE
Include GMPE name when loading layers for gmfs

### ISSUE THREAD

**Issue by CatalinaYepes** [gem/oq-irmt-qgis#410]
When loading layers for Ground Motion Fields, the Realization names appear as "rlz-000", "rlz-001", ...

Include instead the names of the GMPEs, which are stored in the `Realizations` file

**Comment by ptormene** [comment]
GMFs for event-based can't be visualized yet. In that case, we would only have realizations. That's why we are currently letting the user select between realizations and not between GMPEs.

**Comment by ptormene** [comment]
Currently, there is no simple way to extract the names of the GMPEs from the datastore, so I can't proceed on this until this information will be made available engine-side. Therefore, this must be postponed after the workshop.
